src/getHumanAge.test.js

```diff
@@ -7,4 +7,52 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return a correct dog and cats years for an old animals', () => {
+    const result = getHumanAge(100, 100);
+
+    expect(result).toEqual([21, 17]);
+  });
+
+  test('should return zero for 0 array', () => {
```

Comment:
The test description 'should return zero for 0 array' is unclear. Consider rephrasing it to something like 'should return zero for animals with age 0'.